### PR TITLE
YOMA-449 - refactor(About) - About - should correctly display current data

### PR DESCRIPTION
**Pull request by sciphergfx**
## Work Done
Wired biography content to about widget

### CODE REVIEW

File: src/modules/About/About.container.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import selector from 'modules/About/About.selector'
```
Comment (cameronolivier) [inline src/modules/About/About.container.tsx]: please import this locally `import selector from './About.selector'`

File: src/modules/About/About.selector.test.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import * as SUT from './About.selector'
+
+describe('modules/About/About.selector', () => {
+  describe('selector ', () => {
+    it('should select props as expected when available in state', () => {
```
Comment (cameronolivier) [inline src/modules/About/About.selector.test.ts:6]: Add a test for empty state (reflecting user: INITIAL_STATE)

**Review by cameronolivier — CHANGES_REQUESTED** [review]

File: src/modules/About/About.container.tsx (hunk 1 above)
Comment (sciphergfx) [reply src/modules/About/About.container.tsx]: my IDE auto import seem to be causing this issue. I'll make changes, thanks

File: src/modules/About/About.selector.test.ts (hunk 2 above)
Comment (sciphergfx) [reply src/modules/About/About.selector.test.ts:6]: noted

File: src/modules/About/About.container.tsx (hunk 1 above)
Comment (cameronolivier) [reply src/modules/About/About.container.tsx]: I think it's a babel import thing that Ranjit added (that I'd rather remove) - but it's going to affect multiple files - so we need to keep an eye on it

**Review by cameronolivier — DISMISSED** [review]

File: src/modules/User/User.selector.ts [hunk 3]
```diff
@@ -1,3 +1,6 @@
+import { createSelector } from '@reduxjs/toolkit'
+import { propOr } from 'ramda'
 import { RootState } from 'redux/redux.types'
 
 export const selectUser = (state: RootState) => state.user
+export const selectBiography = createSelector(selectUser, propOr('', 'biography'))
```
Comment (cameronolivier) [inline src/modules/User/User.selector.ts:6]: you don't need to use prop or if your initial reducer state is correct - because even if you don't have data, it'll always be an empty string - so what I'd suggest is set 'biography' to '' in the INITIAL_STATE object for the user.

Happy for you to do this in a follow up PR.

**Review by cameronolivier — APPROVED** [review]
Left a comment - please do this in #82
